lib/phoenix/controller/pipeline.ex

```diff
@@ -53,15 +53,15 @@ defmodule Phoenix.Controller.Pipeline do
     cond do
       fallback == nil ->
         raise """
-        action_fallback can only be called when using Phoenix.Controller.
+        action_fallback/1 can only be called when using Phoenix.Controller.
         Add `use Phoenix.Controller` to #{inspect module}
```

Comment:
This might be changed to: 

```elixir
raise """
action_fallback/1 can only be called when using Phoenix.Controller.

Add:

    use Phoenix.Controller

to #{inspect module}
"""

to remove the backtick. Thoughts?